### ISSUE TITLE
19868058 Missing Width

### ISSUE THREAD

**Issue by brotherlogic** [brotherlogic/recordalerting#2672]
From The Cold Jaws Of Prison is missing the width: https://www.discogs.com/madeup/release/4577451
